internal/binder/function/funcs_analytic.go

```diff
@@ -262,7 +262,7 @@ func registerGlobalAggFunc() {
 			}
 			if v1 == nil && v2 == nil {
 				if args[0] == nil || !validData {
-					return 0, true
+					return 0, false
```

Comment:
if when is eval false, shall we return the previous result instead of just 0; And why return false? I expect to return lastResult, true